AppInspector/rules/default/os/dynamic_execution.json

```
@@ -271,5 +271,59 @@
         "confidence": "high"
       }
     ]
+  },
+  {
+    "name": "OS: Dynamic Execution",
+    "id": "AI035520",
+    "description": "OS: Dynamic Execution",
+    "applies_to": [
+      "php"
+    ],
+    "tags": [
+      "OS.Process.DynamicExecution"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "shell_exec",
+        "type": "string",
+        "scopes": [
+          "code"
+        ],
+        "confidence": "high"
+      },
+      {
+        "pattern": "exec",
```

This it too broad.  It would catch regular variable names like `executable`. This should catch only function calls to `exec()`. Maybe this?
```suggestion
        "pattern": " exec[ ]*\\(",
```